Source/Core/DiscIO/VolumeVerifier.h

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "DiscIO/Volume.h"
+
+// To be used as follows:
+//
+// VolumeVerifier verifier(volume);
+// verifier.Start();
+// while (verifier.GetProgress() != verifier.GetMaxProgress())
+//   verifier.Process();
+// verifier.Finish();
+// auto result = verifier.GetResult();
+//
+// Start, Process and Finish may take some time to run.
+//
+// GetResult() can be called before the processing is finished, but the result will be incomplete.
+
+namespace IOS::ES
+{
+class SignedBlobReader;
+}
+
+namespace DiscIO
+{
+class FileInfo;
+
+class VolumeVerifier final
+{
+public:
+  enum class Severity
+  {
+    None,  // Only used internally
```

Comment:
Thoughts on moving this comment _before_ the member? Visual Studio will show that comment in IntelliSense; not sure about other IDEs though.

Reply:
Having one comment at the top of the whole list makes it look a bit ambiguous whether the comment applies to just the first entry or the whole list, so I'd prefer to keep it this way.